common/v2/features/SwapAssets/SwapAssetsFlow.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+
+import { ExtendedContentPanel } from 'v2/components';
+import {
+  SwapAssets,
+  SelectAddress,
+  ConfirmSwap,
+  SwapTransactionReceipt,
+  SetAllowance
+} from './components';
+import { ROUTE_PATHS } from 'v2/config';
+import { ISwapAsset, LAST_CHANGED_AMOUNT } from './types';
```

Comment:
For readability sake please respect the order of import
1. react
2. node_modules
3. */legacy
4. v2
5. relative

Reply:
Updated swap files, but this should be fixed across the whole project.